---
title: "About"
---

## Background

Parkinson’s disease (PD) is the second most prevalent neurodegenerative disease. However, there is a lack of effective treatments able to stop or delay disease progression. The aggregation of a synaptic protein, α-synuclein (aSyn), is the main neurological hallmark of PD and thus, a promising target for therapeutic intervention. Therefore, there is a pressing need to identify and develop novel molecules able to target aSyn aggregates specifically.

Our research group has recently uncovered the physicochemical principles responsible for the binding of short peptides to aSyn toxic species with nanomolar affinity [[1]](references.html). These include alpha-helical propensity, amphipathicity and positive net charge. Such experimentally derived parameters allowed us to identify a human peptide (LL-37) that binds tightly to aSyn aggregates, blocks their neurotoxicity and is believed to play a role in controlling aSyn aggregation in disease-relevant tissues. This discovery opened an avenue for identifying additional human endogenous peptides as therapeutic entities to treat PD or define new biomarkers to monitor in PD patients [[2]](references.html).  We are confident that the previously identified peptides are only a small fraction of the universe of natural peptides harboring such activities in the human peptidome/microbiome.

The aSynPEP database collects human natural peptides (neuropeptides, antimicrobial peptides, gut-microbiome derived peptides and food-derived bioactive peptides) which meet the described criteria, therefore having the potential to selectively bind aSyn toxic oligomers, blocking them and inhibiting their irreversible fibrilization.

## Database information

**ID:** Peptide unique identifier.

**Name:** Peptide name.

**Type**: Peptide type (neuropeptide, antimicrobial, gut microbiome, food-derived).

**Helicity (AGADIR):** Helical score obtained from AGADIR external software [[3]](references.html).

**Amphipaticity (uH):** Hydrophobic dipole moment obtained from an array of hydrophobicity values [[4]](references.html).

**Net charge per residue (NCPR):** Global net charge averaged per residue obtained using the Henderson-Hasselbalch equation at pH 7.

**Hydrophobicity (H):** Hydrophobic score obtained with Fauchere et al. scale (1983).

**Length:** Peptide's number of residues.

**Longest inhibitory region:** Region with inhibitory potential. It can be the full-length peptide or a defined region of it (obtained from 18aa windows ~ 5 helical turns).

**Sequence:** amino acidic sequence of the peptide.

## Record information

In addition to the information collected in the main datatable, each record is enriched with the following additional features/annotations:

**Source ID:** ID of external database (NeuroPep, DRAMP, GMRepo).

**UniProt accession number:** UniProt ID.

**NCBI Taxon ID:** NCBI organism identifier.

**References:** PubMedIDs associated to each peptide.

**Peptide sequence:** Full length peptide with the longest inhibitory region highlighted.

**Helical wheel:** Helical wheel depicts the spatial organization of hydrophobic and charged residues of the inhibitory region in an alpha-helical distribution.

**pI:** Isoelectric point of the peptide inhibitory region. Calculated with localCIDER [[5]](references.html).

**Mass:** Mass of the inhibitory regions (in kDa).

**AlphaFold structure:** Predicted peptide structure by AlphaFold2-ColabFold [[6]](references.html).

**pH of order:** Predicted pH-dependent order-disorder profile provided by DispHred [[7]](references.html).

**Cytotoxicity:** as reported by ToxIBTL [[8]](references.html).

**Blood-brain barrier permeability:** as reported by BBPPred [[9]](references.html).

**Cell-penetrating:** as reported by BchemRF-CPPred [[10]](https://www.nature.com/articles/s41598-021-87134-w)

**Expression:** as reported by The human protein atlas [[11]](references.html) and ProteomicsDB [[12]](references.html).

**Pathways:** as reported by PubChem [[13]](references.html).


## aSynPEP discriminative algorithm

The aSynPEP-DB collects natural peptides that can be found in humans. However, users might be interested in designing synthetic peptides fulfilling the described criteria or testing alternative datasets. For this reason, we have added the possibility of predicting such features with the complementary [aSynPEP descriminative algorithm](http://asynpep.ppmclab.com/).




